client/my-sites/customer-home/main.jsx

```diff
@@ -56,6 +58,16 @@ const Home = ( {
 		);
 	}
 
+	if ( 'purchase-success' === noticeType ) {
+		const reduxDispatch = useDispatch();
```

Comment:
This probably will throw an error because React hooks cannot be used inside a condition. This is partially my fault for not being clear in my suggestion. If you move this line up to line 50 then it should work.

Reply:
I've added a new commit that fixes this 👍